dev-client/src/components/content/info/landpks/LandPKSInfoButton.tsx

```diff
@@ -15,24 +15,15 @@
  * along with this program. If not, see https://www.gnu.org/licenses/.
  */
 
-import {ModalTrigger} from 'terraso-mobile-client/components/modals/Modal';
-import {Heading} from 'terraso-mobile-client/components/NativeBaseAdapters';
-import {OverlaySheet} from 'terraso-mobile-client/components/sheets/OverlaySheet';
+import {AppBarIconButton} from 'terraso-mobile-client/components/buttons/icons/appBar/AppBarIconButton';
+import {LandPKSInfoContent} from 'terraso-mobile-client/components/content/info/landpks/LandPKSInfoContent';
+import {InfoSheet} from 'terraso-mobile-client/components/sheets/InfoSheet';
 
-type Props = {
-  Header: React.ReactNode;
-  trigger?: ModalTrigger;
+export const LandPKSInfoButton = () => {
```

Comment:
Unsure why git decided that this is a rename; it's an all-new component that migrates `LandPKSInfoContent` from its ad-hoc modal container to the new standard `InfoSheet`.